<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainDlg</class>
 <widget class="QDialog" name="mainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="tiffLayout">
      <item>
       <widget class="QLabel" name="labelTiffFileName">
        <property name="text">
         <string>TiffFile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tiffFileName"/>
      </item>
      <item>
       <widget class="QPushButton" name="tiffFileButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <widget class="QCheckBox" name="headerChBox">
        <property name="text">
         <string>Header</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="headerFileName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="headerFileButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="wcsLayout">
      <item>
       <widget class="QCheckBox" name="wcsChBox">
        <property name="baseSize">
         <size>
          <width>13</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>WCS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="wcsFileName">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="wcsFileButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="paramLayout">
      <item>
       <widget class="QCheckBox" name="hFlipChBox">
        <property name="text">
         <string>Horizontal Flip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="vFlipChBox">
        <property name="text">
         <string>Vertical Flip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="negativeChBox">
        <property name="text">
         <string>Negative</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QProgressBar" name="progBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="procButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mainDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mainDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
